testdata/ipfsresourcepinnedintegration.golden

```diff
@@ -1 +1 @@
-{"time_created":1649205656806,"time_pinned":1649205657450,"ipfs_hash":"QmUBzUmJn1dzMQ5ooN8yVvK6TRuTdFnxdkxZi8bQDQ3tuS","state":"queued","size":"125"}
\ No newline at end of file
+{"time_created":1701264369447,"time_pinned":1701273659655,"ipfs_hash":"QmUBzUmJn1dzMQ5ooN8yVvK6TRuTdFnxdkxZi8bQDQ3tuS","state":"queued","size":"125"}
```

Comment:
Both `testdata/ipfsresourcepinnedintegration.golden` and `testdata/ipfsresourcepinnedobjectsintegration.golden` files get modified each time `go test ./...` is ran.  

AFAICT it's the field `"time_pinned"`. Probably linked to the `*update = true` that should probably be dropped:
https://github.com/blockfrost/blockfrost-go/blob/1969aacf5839041434f9dda7784bb9b83a0ab60f/ipfs_test.go/#L78

Reply:
Thank you for looking into this! I commented out the line in 89ad6f2d285624d8cacda28b93b22b833520e8a2